scripts

```diff
@@ -0,0 +1 @@
+.travis
```

Comment:
this was probably added by accident, right?

Reply:
No, this is on purpose. scripts is (currently) a symlink for .travis that can later be pulled out (without much difficulty) into a proper repo for one-off dev/user scripts. All of those scripts are in .travis right now, so I just symlinked it.